sql/expression/function/date.go

```diff
@@ -330,3 +331,170 @@ func (ut *UnixTimestamp) String() string {
 		return "UNIX_TIMESTAMP()"
 	}
 }
+
+// UTCTimestamp converts argument to UTC.
+type UTCTimestamp struct {
```

Comment:
I think you may have the wrong function.
https://dev.mysql.com/doc/refman/8.0/en/date-and-time-functions.html#function_utc-timestamp
`utc_timestamp` should function similarly to `now`, except that it always returns in UTC.